interfaces/automated-testing/scd/design/astm-strategic-coordination/nominal-planning-priority/sequence.plantuml

```diff
@@ -20,9 +20,22 @@ TD <- USS1: 200 Ok - returns Operational Intent 1
 
 ...
 
-== TD injects via Second USS a conflicting Operational Intent 2 in space and/or time with a higher priority ==
+== TD injects via Second USS a conflicting flight 2 in space and/or time with a higher priority ==
 
-TD -> USS2: PUT /inject/flight
+TD -> USS2: PUT /inject/flight\n{"state": "Accepted"}
+TD <- USS2: 200 Ok - Operational Intent 2 created\n{"result": "Planned"}
+
+TD -> DSS: POST /dss/v1/operational_intent_references/query\nwith area_of_interest encompassing Operational Intent 2
+TD <- DSS: 200 Ok - returns Operational Intent Reference 2 with entityid2 managed by Second USS
+
+TD -> USS2: GET /uss/v1/operational_intents/{entityid2}
+TD <- USS2: 200 Ok - returns Operational Intent 2
+
+...
+
+== TD transitions flight in Second USS to Activated ==
```

Comment:
For completeness, would it make sense to try to activate flight one after flight 2 has been planned and before activating flight 2?

Reply:
After some thoughts, that should be subject to a different test and do not have to be done now.

Reply:
Yeah, that corresponds with my thinking.  I think that is a different, valid test, but the ordering here is probably the most interesting (because if we attempted to activate flight 1 first, it might be removed from the system by the time we attempted to activate flight 2, so we wouldn't necessarily test the ability of the USS to activate a high-priority flight intersecting a lower-priority flight).